examples/rvc-app/rvc-common/src/rvc-device.cpp

```diff
@@ -6,11 +6,75 @@ using namespace chip::app::Clusters;
 
 void RvcDevice::Init()
 {
+    mServiceAreaInstance.Init();
     mRunModeInstance.Init();
     mCleanModeInstance.Init();
     mOperationalStateInstance.Init();
+
+    // hardcoded fill of SUPPORTED MAPS for prototyping
+    uint8_t supportedMapId_XX  = 3;
+    uint8_t supportedMapId_YY  = 245;
+
+    mServiceAreaInstance.AddSupportedMap(supportedMapId_XX, CharSpan::fromCharString("My Map XX"));
+    mServiceAreaInstance.AddSupportedMap(supportedMapId_YY, CharSpan::fromCharString("My Map YY"));
+
```

Comment:
Originally these locations were selected to test some of the requirements for one field value vs. another. However, those requirements changed a lot and got more complex.  So I just tried to make them interesting.

They should probably be setup differently for the 'real' example program.

Reply:
I will deal with this in a separate PR.